### PR TITLE
Update the sqlalchemy

### PR DESCRIPTION
**Pull request by zhangjustin**
1. Removed unused import.
2. Update the sqlalchemy version control.
3. Update the zask version to 1.9.5.

Test:
1. Run the unittest

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,7 +37,7 @@
     packages=find_packages(),
     install_requires=[
         'zerorpc==0.5.1',
```
Comment (tjoelsson) [inline setup.py]: Can you also change this to: `zerorpc>=0.5.1, <0.7`?

File: CHANGES [hunk 2]
```diff
@@ -3,6 +3,14 @@ Zask Changelog
 
 Here you can see the full list of changes between each Zask release.
 
+
+Version 1.9.5
+-------------
+released on July 13th 2017
```
Comment (tjoelsson) [inline CHANGES]: Empty line above this one

File: CHANGES [hunk 3]
```diff
@@ -3,6 +3,14 @@ Zask Changelog
...
+
+* Update SQLAlchemy to the version 1.1.11
```
Comment (tjoelsson) [inline CHANGES]: Allow SQLAlchemy up to version 1.1

File: tests/test_sqlalchemy.py [hunk 4]
```diff
@@ -1,11 +1,14 @@
-import os
-import atexit
+# -*- coding: utf-8 -*-
+
 import unittest
+
```
Comment (tjoelsson) [inline tests/test_sqlalchemy.py]: Why separate unittest from other builtin libraries?

**Review by tjoelsson — CHANGES_REQUESTED** [review]

**Comment by zhangjustin** [comment]
updated, please review.

File: tests/test_sqlalchemy.py [hunk 5]
```diff
@@ -1,10 +1,9 @@
 # -*- coding: utf-8 -*-
 
 import unittest
-
 import atexit
-from datetime import datetime
 
+from datetime import datetime
```
Comment (tjoelsson) [inline tests/test_sqlalchemy.py]: put together with other builtin packages above

File: CHANGES [hunk 6]
```diff
@@ -6,9 +6,10 @@ Here you can see the full list of changes between each Zask release.
 
 Version 1.9.5
 -------------
+
 released on July 13th 2017
 
-* Update SQLAlchemy to the version 1.1.11
+* Allow SQLAlchemy up to version 1.1
```
Comment (tjoelsson) [inline CHANGES]: Allow ZeroRPC up to version 0.6

**Review by tjoelsson — CHANGES_REQUESTED** [review]

File: tests/test_sqlalchemy.py (hunk 5 above)
Comment (zhangjustin) [reply tests/test_sqlalchemy.py]: sorry, forget that datetime is builtin package

**Comment by zhangjustin** [comment]
Updated, please review.

File: tests/test_sqlalchemy.py [hunk 7]
```diff
@@ -2,8 +2,8 @@
 
 import unittest
 import atexit
-
 from datetime import datetime
```
Comment (tjoelsson) [inline tests/test_sqlalchemy.py]: ? order alphabetically

**Review by tjoelsson — APPROVED** [review]

**Review by mark-juwai — APPROVED** [review]

**Review by tjoelsson — APPROVED** [review]

**Comment by zhangjustin** [comment]
Merge

**Pull request merged**